src/main/Extensions/VSCode/VSCodeActionHandler.ts

```diff
@@ -0,0 +1,13 @@
+import type { ActionHandler } from "@Core/ActionHandler";
+import type { SearchResultItemAction } from "@common/Core";
+import { exec } from "child_process";
+
+export class VSCodeActionHandler implements ActionHandler {
+    public readonly id = "VSCodeHandler";
+
+    public constructor() {}
+
+    public async invokeAction(action: SearchResultItemAction) {
+        exec(`code ${action.argument}`);
+    }
+}
```

Comment:
I suggest using the `CommandlineUtility` instead of `child_process`:

```typescript
import type { ActionHandler } from "@Core/ActionHandler";
import type { CommandlineUtility } from "@Core/CommandlineUtility";
import type { SearchResultItemAction } from "@common/Core";

export class VSCodeActionHandler implements ActionHandler {
    public readonly id = "VSCodeHandler";

    public constructor(private readonly commandlineUtility: CommandlineUtility) {}

    public async invokeAction(action: SearchResultItemAction) {
        await this.commandlineUtility.executeCommand(`code ${action.argument}`);
    }
}
```